refactoring/initial_two_classes.py

```diff
@@ -32,17 +32,32 @@ def add_person(self, name, age, job):
 
     def number_of_connections(self, name):
         """Find the number of connections that a person in the group has"""
-        pass
+        if name in self.connections:
```

Comment:
I'm not sure that this works - which is probably our bad for not having a test of this in the end. I think what you'll need is
```suggestion
        if self.contains(name):
```